workers/unity/Packages/com.improbable.gdk.tools/ShowProgressBarScope.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using UnityEditor;
+
+namespace Improbable.Gdk.Tools
+{
+    internal struct ShowProgressBarScope : IDisposable
```

Comment:
This is a bit odd. You cannot actually nest scopes with this, as `ClearProgressBar` will close the one progress bar that `DisplayProgressBar` reuses.

Reply:
Yep, but it makes the showing/restoring of the progress window exception-proof and removes the need to do a `try/finally`. I don't mind removing it if you like, though.

Reply:
I think it'll be fine.
Here is to hoping unity will someday make a better system for displaying work in progress